File: src/main/java/com/matchFit/post/dto/response/GetMyPostApplicants.java
package com.matchFit.post.dto.response;

import java.util.List;
import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class GetMyPostApplicants {
    private final List<GetMyPostApplicant> applicants;

    public static GetMyPostApplicants from(List<GetMyPostApplicant> applicantDtos) {
        return new GetMyPostApplicants(applicantDtos);
    }
}

File: src/main/kotlin/com/matchFit/post/dto/response/GetMyPostApplicants.kt
package com.matchFit.post.dto.response

data class GetMyPostApplicants(
    val applicants: List<GetMyPostApplicant>
) {
    companion object {
        fun from(applicantDtos: List<GetMyPostApplicant>): GetMyPostApplicants {
            return GetMyPostApplicants(applicantDtos)
        }
    }
}
